packages/nx/src/command-line/show/command-object.ts

```diff
@@ -175,10 +174,9 @@ const showProjectCommand: CommandModule<NxShowArgs, ShowProjectOptions> = {
         'View project information for my-app in the browser'
       ),
   handler: async (args) => {
-    const exitCode = await handleErrors(args.verbose as boolean, async () => {
+    process.exitCode = await handleErrors(args.verbose as boolean, async () => {
       const { showProjectHandler } = await import('./project');
       await showProjectHandler(args);
     });
-    process.exit(exitCode);
```

Comment:
This could have issues with the command hanging, all of our other commands have a `process.exit` for that reason